src/redux/topMovers.ts

```diff
@@ -97,12 +96,13 @@ export const updateTopMovers = (message: ZerionAssetInfoResponse) => (
 ) => {
   const { nativeCurrency } = getState().settings;
   const orderByDirection = message.meta.order_by['relative_changes.1d'];
-  const assets = map(message.payload.info, ({ asset }) => {
+  const assets = message.payload.info.map(({ asset }) => {
```

Comment:
I don't trust the backend and we're not doing any validation so...
```suggestion
  const assets = message?.payload?.info?.map(({ asset }) => {
```